python/ambassador/envoy/v3/v3listener.py

```diff
@@ -990,13 +1045,25 @@ def finalize_http(self) -> None:
             self._filter_chains.append(filter_chain)
 
     def as_dict(self) -> dict:
-        listener = {
+        listener: dict = {
```

Comment:
This is a weird type annotation to add. It can plainly see that it's a dict without an annotation; the reason you'd add the annotation is to tell it the types of the keys and values in the dict; something like `Dict[str, Any]` (or preferably something more specific than `Any`, but looking at it it seems to me that there are at least 4 different types, so... that `Union[…]` would be a little large).

Reply:
Not a blocker, it's just bizarre.